internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -40,6 +41,10 @@ var (
 
 	errNoContainerPath = errors.New(`"path" cannot be empty`)
 	errNoSourceVolume  = errors.New(`"source_volume" cannot be empty`)
+
+	errUIDWithNonManagedFS = errors.New("UID and GID cannot be specified with non-managed EFS")
+	errInvalidUIDGIDConfig = errors.New("set managed filesystem access point creation info: must specify both UID and GID, or neither")
+	errReservedUID         = errors.New("set managed filesystem access point creation info: UID must not be 0")
```

Comment:
Hmm why do we want to wrap this error? Maybe just
```suggestion
	errInvalidUIDGIDConfig = errors.New("must specify both UID and GID, or neither")
	errReservedUID         = errors.New("UID must not be 0")
```

Reply:
Will address in the refactor PR.